doc/source/whatsnew/v0.20.0.txt

```diff
@@ -585,3 +585,4 @@ Bug Fixes
 - Bug in ``Series.replace`` and ``DataFrame.replace`` which failed on empty replacement dicts (:issue:`15289`)
 - Bug in ``pd.melt()`` where passing a tuple value for ``value_vars`` caused a ``TypeError`` (:issue:`15348`)
 - Bug in ``.eval()`` which caused multiline evals to fail with local variables not on the first line (:issue:`15342`)
+- Bug in ``.rank()`` rank incorrectly orders ordered categories
```

Comment:
don't put this at the end, instead use a blank space that I have left above. Add the issue number at the end.